hermes/probe/delete/delete_test.go

```diff
@@ -144,7 +145,7 @@ func TestDeleteRandomFile(t *testing.T) {
 	testProbeName := "testDelete1"
 	ctx := context.Background()
 
-	client := setupFakeSystem(ctx, t)
+	client := fakeStorageClient(ctx, t)
```

Comment:
Optional: I think it would be cleaner if `fakeStorageClient()` was called `populateBucket` or `createTestFiles` (or something to that effect) and it only did the population. Then the test code would look like this:

```
client := fakeclient.New()
if err := createTestFiles(t, client); err != nil {
   ...
}
```

Reply:
I've changed this to `createTestFiles()` and updated the mock package I'm using to the latest in Alicja's PR, so it's now `fakegcs`.